packages/web3-eth/package.json

```diff
@@ -53,7 +53,6 @@
 		"setimmediate": "^1.0.5",
 		"web3-core": "^4.0.1-alpha.5",
 		"web3-errors": "^0.1.1-alpha.4",
-		"web3-eth-accounts": "^4.0.1-alpha.5",
```

Comment:
This should be reverted as  `privateKeyToAddress` from `web3-eth-accounts` [transaction_builder.ts](https://github.com/web3/web3.js/blob/4cecc7a92cd2054c1fd3a57e8da6c1f8da078e38/packages/web3-eth/src/utils/transaction_builder.ts#L41)

Reply:
Thanks for the note. 
Interestingly, the build succeeded but lint detected that.